content-services/content-sources-backend
# Fixes 1876: add endpoint to list public repos

## Summary
Adds a `GET .../v1/public_repositories/` endpoint to return a paginated list all the "public" repositories (i.e. the repositories on the repositories table where "public" is true). These are also the ones used by image builder.
## Testing steps
1. Run `make repos-import` to make sure you've imported all the public repos
2. GET `http://localhost:8000/api/content-sources/v1/public_repositories/?limit=100&offset=0`. Also try using the offset and limit params.
3. You should get a paginated list of results.
